cloudify-manager-worker/templates/cloudify-config.yaml

```diff
@@ -29,21 +29,33 @@ data:
             default: {{ .Values.queue.host }}.{{.Release.Namespace}}.svc.cluster.local
 
     postgresql_client:
+      {{- if .Values.db.useExternalDB }}
+      host: {{ .Values.db.host }}
+      {{- else }}
       host: {{ .Values.db.host }}.{{.Release.Namespace}}.svc.cluster.local
+      {{- end }}
       server_db_name: {{ .Values.db.serverDBName }}
       server_username: {{ .Values.db.serverUsername }}
       server_password: {{ .Values.db.serverPassword }}
       cloudify_db_name: {{ .Values.db.cloudifyDBName }}
       cloudify_username: {{ .Values.db.cloudifyUsername }}
       cloudify_password: {{ .Values.db.cloudifyPassword }}
+      {{- if .Values.db.useExternalDB }}
+      ca_path: {{ .Values.config.postgresqlCaPath }}
+      {{- else }}
       ca_path: {{ .Values.config.caCertPath }}
+      {{- end }}
       ssl_enabled: true
       ssl_client_verification: true
     postgresql_server:
       postgres_password: {{ .Values.db.serverPassword }}
+      {{- if .Values.db.useExternalDB }}
```

Comment:
Eric, I think we should add the cert & key path in case we use external DB as well.

Reply:
So, depending on how the external db is deployed, we will have different levels of required keys/certificates.
Would the approach be to set up some more conditionals, and only add the lines that have defined values?